pkg/utils/helm/utils.go

```diff
@@ -115,9 +115,7 @@ func CheckRelease(name, namespace string) (exist bool, err error) {
 	return exist, err
 }
 
-/*
-* Delete Release given name and namespace
- */
+// DeleteRelease given name and namespace
```

Comment:
DeleteRelease deletes release  with the name and namespace

Reply:
Thanks,and I changed the comment just now.